style.css

```diff
@@ -0,0 +1,66 @@
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.font-style {
+  font-family: 'Poppins', sans-serif;
+}
+
+.color {
+  color: #7f8cff;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+}
+
+a {
+  text-decoration: none;
+}
+
+.logo-text {
+  font-weight: 700;
+  color: #6070ff;
+}
+
+.menu-icon {
+  cursor: pointer;
+}
+
+.container {
+  width: 100%;
+  height: 100vh;
+  background-image: url('/assets/images/header-shapes\ mobile@2x.png');
```

Comment:
You did great achieving the headline 🥳.
unfortunately, the background design doesn't match the Figma design, this is likely because of the repeating background image.
**Your design**
![Screenshot 2022-06-29 at 12 54 57](https://user-images.githubusercontent.com/31113416/176408682-e6501e44-850b-4c02-b153-c01840f80cbe.png)


**Figma design**

![headline-mobile](https://user-images.githubusercontent.com/31113416/176407902-2161288c-963f-4324-95c1-e853c71c08f0.png)

- [ ] Kindly update the  Headline background image to match the Figma design